.github/workflows/build_documentation.yml

```diff
@@ -14,6 +14,6 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: en es fr he pt ru th tr
+      languages: en es fa fr he pt ru th tr
```

Comment:
FYI I've added Farsi to the list of languages here so we can build the docs in the CI